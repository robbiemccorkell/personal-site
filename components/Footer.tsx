import ContentWrapper from "./ContentWrapper";

const Footer = () => (
  <div className="footer">
    <ContentWrapper>
      <div className="inner">
        <div className="socials">
          <a href="https://github.com/robbiemccorkell" target="_blank">
            <img src={require("./icons/github.svg")} />
          </a>
          <a href="https://twitter.com/robbiemccorkell" target="_blank">
            <img src={require("./icons/twitter.svg")} />
          </a>
          <a href="https://www.instagram.com/robbiemccorkell/" target="_blank">
            <img src={require("./icons/instagram.svg")} />
          </a>
        </div>
      </div>
    </ContentWrapper>
    <style jsx>{`
      .footer {
        background-color: #eee;
      }
      .inner {
        height: 80px;
        display: flex;
        justify-content: flex-end;
        align-items: center;
      }
      .socials a {
        margin-left: 14px;
      }
    `}</style>
  </div>
);

export default Footer;
