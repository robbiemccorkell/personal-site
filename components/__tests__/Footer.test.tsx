import { render } from "@testing-library/react";
import Footer from "../Footer";

test("matches snapshot", () => {
  const output = render(<Footer />);

  expect(output.container).toMatchSnapshot();
});
