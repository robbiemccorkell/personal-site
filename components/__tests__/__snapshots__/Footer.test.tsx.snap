// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <div
    class="jsx-391651169 footer"
  >
    <div
      class="jsx-985212870 wrapper"
    >
      <div
        class="jsx-391651169 inner"
      >
        <div
          class="jsx-391651169 socials"
        >
          <a
            class="jsx-391651169"
            href="https://github.com/robbiemccorkell"
            target="_blank"
          >
            <img
              class="jsx-391651169"
              src="file"
            />
          </a>
          <a
            class="jsx-391651169"
            href="https://twitter.com/robbiemccorkell"
            target="_blank"
          >
            <img
              class="jsx-391651169"
              src="file"
            />
          </a>
          <a
            class="jsx-391651169"
            href="https://www.instagram.com/robbiemccorkell/"
            target="_blank"
          >
            <img
              class="jsx-391651169"
              src="file"
            />
          </a>
        </div>
      </div>
    </div>
  </div>
</div>
`;
